packages/composites/token-allocation/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,34 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+import { makeConfig } from '../../src/config'
+
+describe('execute', () => {
+  const jobID = '1'
+  process.env.DATA_PROVIDER_URL = 'ignoreable'
```

Comment:
Same - Only want to fill this in if it isn't provided

```suggestion
  if (!process.env.DATA_PROVIDER_URL) process.env.DATA_PROVIDER_URL = 'ignoreable'
```

Reply:
We should make this a separate PR if we can, i tried to avoid actual code changes as much as possible